tests/bin/expected/petstore.ts

```diff
@@ -403,11 +403,11 @@ export interface paths {
         /**
          * Invalid username supplied
          */
-        '400': any
+        '400': unknown
         /**
          * User not found
          */
-        '404': any
+        '404': unknown
```

Comment:
👍🏻 I’m OK with this change. Really, any undeclared type should be “dangerous” and I do like that this will surface more errors than `any`